apps/opencs/view/render/lightingnight.cpp

```diff
@@ -20,6 +20,9 @@ void CSVRender::LightingNight::activate (osg::Group* rootNode)
     mLightSource->setLight(light);
 
     mRootNode->addChild(mLightSource);
+
+    // In the light mode windows in exterior cells should glow.
```

Comment:
I don't understand this comment. Well, I understand the meaning behind it, but "the light mode" doesn't make much sense without the context.